recipes/aws-checksums/all/conanfile.py

```diff
@@ -0,0 +1,71 @@
+from conans import CMake, ConanFile, tools
+import os
+
+
+class AwsChecksums(ConanFile):
+    name = "aws-checksums"
+    description = "Cross-Platform HW accelerated CRC32c and CRC32 with fallback to efficient SW implementations. C interface with language bindings for each of our SDKs "
+    topics = ("conan", "aws", "checksum", )
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/awslabs/aws-checksums"
+    license = "Apache-2.0",
+    exports_sources = "CMakeLists.txt", "patches/**"
+    generators = "cmake"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        del self.settings.compiler.cppstd
+        del self.settings.compiler.libcxx
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_folder = "aws-checksums-{}".format(self.version)
+        os.rename(extracted_folder, self._source_subfolder)
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.definitions["BUILD_TESTING"] = False
+        self._cmake.definitions["CMAKE_MODULE_PATH"] = self.build_folder.replace("\\", "/")
```

Comment:
why is this required?

Reply:
I don't remember, but it looks like it can be removed.